wrapanapi/systems/virtualcenter.py

```diff
@@ -263,63 +426,42 @@ def _clone(self, destination, resourcepool=None, datastore=None, power_on=True,
 
         source_template = self.raw
 
-        vm_clone_spec = vim.VirtualMachineCloneSpec()
-        vm_reloc_spec = vim.VirtualMachineRelocateSpec()
         # DATASTORE
         if isinstance(datastore, six.string_types):
-            vm_reloc_spec.datastore = self.system.get_obj(vim.Datastore, name=datastore)
-        elif isinstance(datastore, vim.Datastore):
-            vm_reloc_spec.datastore = datastore
-        elif datastore is None:
+            picked_datastore = self.system.get_datastore(datastore_name=datastore)
+        elif isinstance(datastore, (vim.Datastore, vim.StoragePod)):
```

Comment:
Q: So `datastore` can be a vim.Datastore or a vim.StoragePod? Thoughts on having the cluster be a separate argument?

Reply:
Discussed offline. Keeping current implementation.